plugin/rocks.lua

```diff
@@ -24,11 +24,12 @@ log.trace("loading rocks.adapter")
 local adapter = require("rocks.adapter")
 log.trace("loading rocks config")
 local config = require("rocks.config.internal")
-log.debug("Using luarocks config " .. config.luarocks_config)
 
 -- Initialize the luarocks loader
 if config.enable_luarocks_loader then
-    require("rocks.loader").enable()
+    nio.run(function()
+        require("rocks.loader").enable()
+    end)
```

Comment:
question: is there ever a situation where this could break certain configurations that try to make use of the loader's features directly on startup? If there's a possibility of a race condition where the loader is enabled after it is used then we might want to reconsider making this behaviour asynchronous.

Reply:
I have been wondering about this too.

There is some strange behaviour I observed in the integration test: 

```bash
nvim -u .github/resources/init-integration.lua -c +wq
```

now exits with a non-zero exit code. But it does what it's supposed to do.
My guess is that it reaches `+q` before it finishes writing the luarocks config. 

I think it's safer not to make it async.

Reply:
Done.